### PR TITLE
Update onboard-configure.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated permissions needed to ONboard devices to the server

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9265

File: microsoft-365/security/defender-endpoint/onboard-configure.md [hunk 1]
```diff
@@ -37,6 +37,18 @@ Deploying Microsoft Defender for Endpoint is a two-step process.
 
 :::image type="content" source="images/deployment-steps.png" alt-text="The onboarding and configuration process" lightbox="images/deployment-steps.png":::
 
+## Role-based access control
+
+Microsoft recommends using Privileged Identity Management to manage your roles to provide additional auditing, control, and access review for users with directory permissions.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/onboard-configure.md]:
```suggestion
We recommend using Privileged Identity Management to manage your roles to provide additional auditing, control, and access review for users with directory permissions.
```

File: microsoft-365/security/defender-endpoint/onboard-configure.md [hunk 2]
```diff
@@ -37,6 +37,18 @@ Deploying Microsoft Defender for Endpoint is a two-step process.
...
+
+Defender for Endpoint supports two ways to manage permissions:
+
+- **Basic permissions management**: Set permissions to either full access or read-only. Users with Global Administrator or Security Administrator roles in Azure Active Directory have full access. The Security reader role has read-only access and does not grant access to view machines/device inventory.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/onboard-configure.md]:
```suggestion
- **Basic permissions management**: Sets permissions to either full access or read-only. Users with global administrator or security administrator roles in Azure Active Directory (Azure AD) have full access. The security reader role has read-only access and does not grant access to view machines/device inventory.
```

File: microsoft-365/security/defender-endpoint/onboard-configure.md [hunk 3]
```diff
@@ -37,6 +37,18 @@ Deploying Microsoft Defender for Endpoint is a two-step process.
...
+
+- **Role-based access control (RBAC)**: Set granular permissions by defining roles, assigning Azure AD user groups to the roles, and granting the user groups access to device groups. For more information. see [Manage portal access using role-based access control](rbac.md).
+
+Microsoft recommends leveraging RBAC to ensure that only users that have a business justification can access Defender for Endpoint.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/onboard-configure.md]:
```suggestion
We recommend leveraging RBAC to ensure that only users that have a business justification can access Defender for Endpoint.
```

File: microsoft-365/security/defender-endpoint/onboard-configure.md [hunk 4]
```diff
@@ -37,6 +37,18 @@ Deploying Microsoft Defender for Endpoint is a two-step process.
...
 
+## Role-based access control
+
+Microsoft recommends using Privileged Identity Management to manage your roles to provide additional auditing, control, and access review for users with directory permissions.
+
+Defender for Endpoint supports two ways to manage permissions:
+
+- **Basic permissions management**: Set permissions to either full access or read-only. Users with Global Administrator or Security Administrator roles in Azure Active Directory have full access. The Security reader role has read-only access and does not grant access to view machines/device inventory.
+
+- **Role-based access control (RBAC)**: Set granular permissions by defining roles, assigning Azure AD user groups to the roles, and granting the user groups access to device groups. For more information. see [Manage portal access using role-based access control](rbac.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/onboard-configure.md]:
```suggestion
- **Role-based access control (RBAC)**: Sets granular permissions by defining roles, assigning Azure AD user groups to the roles, and granting the user groups access to device groups. For more information. see [Manage portal access using role-based access control](rbac.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**